.cargo/config.toml

```diff
@@ -1,2 +1,2 @@
 [alias]
-clippy-ci = "clippy --all-features --all-targets --all"
+clippy-ci = "clippy --all-features --lib --all"
```

Comment:
I think we should maybe have a second alias that's only used for MSRV? For stable etc we should keep checking `--all-targets`.

Reply:
I introduced a second alias for the MSRV use-case.